.env.template

```diff
@@ -2,6 +2,7 @@ GITHUB_TOKEN = ""
 TIMEZONE = "Etc/UTC"
 CSV_COLUMN_NUMBER_FOR_GITHUB_ID = ""
 CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID = ""
+IGNORE_OWN_EVENTS = true
```

Comment:
let's change this to IGNORE_SELFOWNED_EVENTS because that's a little bit clearer. (And then, change the variable name everywhere else it's used- the README and in index.js)

And let's set it equal to an empty string "" because this is an option and not the default functionality for Starfish.